.eslintrc.js

```diff
@@ -9,13 +9,12 @@ module.exports = {
   },
   extends: [
     'standard',
-    'plugin:vue/recommended'
+    'plugin:vue/recommended',
+    'plugin:prettier/recommended'
```

Comment:
what does this part do that the standard part doesn't?

Reply:
in reference to the `'plugin:prettier/recommended'` line.

Reply:
If that is answered, i will merge.

Reply:
Hoping [this section](https://github.com/prettier/eslint-plugin-prettier#recommended-configuration) helps of the read me for the plugin. But it's basically suggested for the best way to have eslint and prettier play nicely together. Otherwise seems like you'll have 1 off conflicts between the two resulting in linting errors.

Reply:
I'm accepting this PR.